source/crud/read-operations/limit.txt

```diff
@@ -0,0 +1,80 @@
+====================================
+Limit the Number of Returned Results
+====================================
+
+.. default-domain:: mongodb
+
+Use limit to cap the number of documents that can be returned from a
+read operation. Limit functions as a cap on the maximum number of
+documents that the operation can return, but the operation can return
+a smaller number of documents if there are not enough documents present
+to reach the limit. If limit is used with the
+:doc:`skip </crud/read-operations/skip>` method, the skip applies
+first and the caponly applies to the documents left over after
+the skip.
+
+Consider a collection containing documents that describe books. To
+insert this data into a collection, run the following operation:
+
+.. code-block:: javascript
+
+   await books.insertMany([
+     { "_id": 1, "name": "The Brothers Karamazov", "author": "Dostoyevsky", "length": 824 },
+     { "_id": 2, "name": "Les Misérables", "author": "Hugo", "length": 1462 },
+     { "_id": 3, "name": "Atlas Shrugged", "author": "Rand", "length": 1088 },
+     { "_id": 4, "name": "Infinite Jest", "author": "Wallace", "length": 1104 },
+     { "_id": 5, "name": "Cryptonomicon", "author": "Stephenson", "length": 918 },
+     { "_id": 6, "name": "A Dance With Dragons", "author": "Tolkein", "length": 1104 },
+   ]);
+
+Pass the following sort document to a read operation to ensure that the
+operation returns books with longer lengths before books with shorter
+lengths. Applying a limit of 3 to this operation causes this operation
```

Comment:
Nitpick: maintain the imperative voice here for parallel structure among the sentences.
"Pass the following sort document..."
"Apply a limit of `3` to return only the three longest books..."